lib/Smokeping.pm

```diff
@@ -3916,6 +4088,15 @@ sub load_cfg ($;$) {
                die "ERROR: Could not load Storable Support. This is required for the Charts feature - $@\n" if $@;
            load_sorters $cfg->{Presentation}{charts};
         }
+        #initiate a connection to InfluxDB (if needed)
+        if(! defined $influx && defined $cfg->{'InfluxDB'}{'host'}){
```

Comment:
```suggestion
        if (not defined $influx and defined $cfg->{InfluxDB}{host}){
```

Reply:
This was removed entirely with this commit: https://github.com/oetiker/SmokePing/pull/215/commits/c4076fcdf9d3ca722df5dbe121bd1a77fac621c7